charts/influxdb2/templates/ingress.yaml

```diff
@@ -1,30 +1,42 @@
 {{- if .Values.ingress.enabled -}}
+{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1" }}
```

Comment:
Could you please change the condition to check for Ingress object in v1 since some k8s versions have v1 API but that does include Ingress (it's since 1.19)? Thank you.

`{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1/Ingress" }}`

Reply:
added.